mobile-first/css/styles.css

```diff
@@ -0,0 +1,449 @@
+    width: 800px;
+    max-width: 880px;
+  }
+
+  .container .navcontainer .logo {
+    padding: 0;
+  }
+
+  .container .navcontainer .navmenu {
+    display: flex;
+    flex-direction: row;
+    padding-left: 30px;
+    justify-content: space-between;
+    width: 92%;
+  }
+
+  .container .navcontainer .navmenu a {
+    border: none;
+    font-size: 1em;
+  }
+
+  #menu-links {
+    display: flex;
+  }
+
+  .container .navcontainer .burger {
+    display: none;
+  }
+
+  /* above image content*/
+
+  .container .above-img {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 28px 30px 5px 30px;
+  }
+
+  .container .above-img .above-content-block {
+    width: 50%;
+    padding-bottom: 0;
+  }
+  
+  .container .above-img .featuress {
+    padding-bottom: 46px;
+  }
+  
+  .container .above-img .about {
+    padding-left: 10px;
+    padding-bottom: 0;
+  }
+
+  .container .middle-img {
+
+  }
+  
+  /* below image content */
+  .container .below-img {
+    padding: 0 0 0 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .container .below-img .below-item {
+    width: 33%;
+    padding-left: 30px;
+    padding-bottom: 44px;
+  }
+  
+  .container .below-img .product {
+      padding:0 20px;
+  }
+  
+  .container .below-img .vision {
+    padding: 0 10px;
+  }
+  
+  .container address {
+    padding: 25px 10px 0 30px;
+  }
+  
+  footer {
+    padding-top: 25px;
+    text-align: center;
+  }
+
+}
+
+@media (min-width: 880px) {
+  body {
+    margin: 0 auto;
+    max-width: 880px;
+    display: block;
+    position: relative;
+  }
+  /* navbar */
+
+  .container nav {
+    width: 880px;
+  }
+
+  .container .navcontainer {
+    flex-direction: row;
+    justify-content: space-between
+  }
+
+  .container .navcontainer .navmenu {
+    width: 69%;
+    padding-left: 0;
+    margin: none;
+  }
+
+  /* header content */
+
+  .container .headermain {
+    display: flex;
+    justify-content: space-between
+  }
+
+  .container .headermain img {
+    display: block;
+  }
+
+  .container .headermain .headertext {
+    padding-left: 127px;
+  }
+
+  /* above image content*/
+
+  #above-img {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 27px 0 0 0;
+  }
+
+  .container .above-img .above-content-block {
+    width: 50%;
+    padding: none;
+  }
+  
+  .container .above-img .featuress {
+    padding-left: 0;
+  }
+  
+  .container .above-img .about {
+    padding-left: 0;
+  }
+
+  .container .middle-img {
+    padding: 0 0 0 0;
+    margin-top: -14px;
+  }
+  
+  /* below image content */
+  .container .below-img {
+    padding-top: 28px;
+  }
+
+  .container .below-img .below-item {
+    width: 33%;
+    padding-left: 0;
+    padding-bottom: 30px;
+  }
+  
+  .container .below-img .product {
+      padding-left: 2px;
+  }
+  
+  .container .below-img .vision {
+    padding-left: 6px;
+  }
+  
+  .container address {
+    padding: 25px 0 0 0;
```

Comment:
you can make styles like this cleaner by making it padding-top , bottom, left, right etc. Then you can leave out the zeros